python/tidy.py

```diff
@@ -346,6 +346,8 @@ def check_rust(file_name, lines):
             for match in re.finditer(pattern, line):
                 if not filter_func(match, line):
                     continue
+
+                print line
```

Comment:
Oops...this will print every erroneous line. Debugging garbage made it into my branch. amended, force pushed, so that crap isn't even in history.

Thank you for catching that...